<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"/>
<style>body{background-color:white;}</style>
<script src="datatables_libs/htmlwidgets/htmlwidgets.js"></script>
<script src="datatables_libs/jquery/jquery.min.js"></script>
<link href="datatables_libs/datatables-css/datatables-crosstalk.css" rel="stylesheet" />
<script src="datatables_libs/datatables-binding/datatables.js"></script>
<script src="datatables_libs/pymjs/pym.v1.min.js"></script>
<link href="datatables_libs/dt-core/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="datatables_libs/dt-core/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="datatables_libs/dt-core/js/jquery.dataTables.min.js"></script>
<link href="datatables_libs/crosstalk/css/crosstalk.css" rel="stylesheet" />
<script src="datatables_libs/crosstalk/js/crosstalk.min.js"></script>
  <title>datatables</title>
</head>
<body>
<div id="htmlwidget_container">
  <div id="htmlwidget-d000429013698f738f6e" style="width:100%;height:800px;" class="datatables html-widget"></div>
  <script>HTMLWidgets.pymChild = new pym.Child();HTMLWidgets.addPostRenderHandler(function(){
                                setTimeout(function(){HTMLWidgets.pymChild.sendHeight();},100);
                            });</script>
</div>
<script type="application/json" data-for="htmlwidget-d000429013698f738f6e">{"x":{"filter":"none","fillContainer":true,"data":[["2020-02-06","2020-02-06","2020-02-06","2020-02-05","2020-02-05","2020-02-04","2020-02-04","2020-02-04","2020-02-04","2020-02-04","2020-02-04","2020-02-03","2020-02-03","2020-02-01","2020-01-31","2020-01-31","2020-01-30","2020-01-30","2020-01-30","2020-01-30","2020-01-29","2020-01-29","2020-01-29","2020-01-28","2020-01-28","2020-01-28","2020-01-28","2020-01-28","2020-01-27","2020-01-27","2020-01-27","2020-01-27","2020-01-27","2020-01-27","2020-01-25","2020-01-25","2020-01-25","2020-01-24","2020-01-24","2020-01-23","2020-01-23","2020-01-23","2020-01-23","2020-01-22","2020-01-22","2020-01-22","2020-01-22","2020-01-20","2020-01-20","2020-01-19","2020-01-18","2020-01-18","2020-01-18","2020-01-17","2020-01-16","2020-01-16","2020-01-16","2020-01-15","2020-01-15","2020-01-15","2020-01-15","2020-01-15","2020-01-15","2020-01-14","2020-01-14","2020-01-13","2020-01-12","2020-01-10","2020-01-10","2020-01-09","2020-01-09","2020-01-09","2020-01-08","2020-01-08","2020-01-08","2020-01-07","2020-01-07","2020-01-07","2020-01-07","2020-01-07","2020-01-07","2020-01-07","2020-01-07","2020-01-07","2020-01-06","2020-01-06","2020-01-06","2020-01-06","2020-01-06","2020-01-06","2020-01-06","2020-01-05","2020-01-04","2020-01-04","2020-01-04","2020-01-03","2020-01-02","2020-01-02","2020-01-02","2020-01-02"],["timoast","steverozen","candiceaman","SafiaDanovi","rlmcelreath","AliciaOshlack","prof_mirya","guptalabunc","vboykis","rlmcelreath","LGMartelotto","mikelove","LGMartelotto","thisisthethirty","Emil_Hvitfeldt","candiceaman","satijalab","SusannaLHarris","EpgntxEinstein","chuurveg","DNATech_Lutz","AllenDowney","BiotechMongoose","EvaMaeRey","DGautheret","lpachter","MollyGasp","rddowell","TiffanyCheingHo","markrobinsonca","tangming2005","michelebusby","AlbertVilella","OmicsOmicsBlog","nevillesanjana","vboykis","JennyBryan","CedScherer","gungulati","aaronquinlan","dnusinow","edyong209","TerryTangYuan","jozefhajnala","basic_scientist","dataandme","LGMartelotto","kushaljoshi22","Emil_Hvitfeldt","LGMartelotto","tangming2005","JswLab","mtliendo","tangming2005","WyattMcDonnell","jeremyphoward","VicBergerIV","not_a_reptiloid","vboykis","rtelmore","hoheyn","jokergoo","LGMartelotto","CoulSim","JEFworks","CFlensburg","LGMartelotto","ivivek87","tangming2005","matthewdwood82","jkbren","eswartz","mansfieldnstuff","rlmcelreath","ksonpham","michael_laub8","pnatarajanmd","bradleybio","mansfieldnstuff","TheStephLocke","elbersb","seandavis12","falexwolf","heymanitshayden","ariellesallai","ParkLabGenomics","ematsen","dataandme","ctmurphy1","hoheyn","bioturing","semiramis_cj","NCIEytanRuppin","biorxivpreprint","buttpoems","drob","David_McGaughey","stiles","topepos","LukeMMoney"],["Remember to tune in tomorrow for the SCGD rebroadcast! This will be running Friday 7th Feb from 10:30 am - 6:00 pm Perth time (AWST), 12:30 pm - 8:00 pm in Brisbane (AEST), 1 pm - 8:30 pm in Adelaide (ACDT), and 1:30 pm - 9:00 pm in Melbourne/Sydney (AEDT) https://t.co/ZlMLGiAjbd https://t.co/GE0zV0CFSi","Author of \"The Repertoire of Mutational Signatures in Human Cancer\" in Nature. Read here: https://t.co/Pljzp9dhlu","thankfully @stchlk is a member of data science twitter so I am the beneficiary of some gems like this. https://t.co/d2JcptVZSL","Pan-Cancer Analysis of Whole Genomes. A collection of research and related content from the ICGC/TCGA consortium on whole&amp;amp;#8209;genome sequencing and integrative analysis of cancer https://t.co/YgoDAXPOg7","Slides + audio: A gentle 2 hour introduction to Bayesian data analysis &amp;amp; causal inference. This is \"gentle\" because it ignores computation &amp;amp; focuses instead on motivation, basics of Bayesian updating, simple confounds &amp;amp; colliders. https://t.co/1EJDDIqWo0","New paper from PhD student Andreas Halman evaluating short tandem repeat genotyping tools in exome data.\n\nhttps://t.co/ibnqRzMiiz\n\nConstructive comments and criticisms welcome.","In Jan 2018, I set goal to email someone 1x a week to tell them how I liked a book or article. \n\nI sent my 100th note today. It is one of my favorite things. \n\nNext time you read something interesting, useful, or thought-provoking, why not tell the author?","How do cancers develop rampant chromosomal instability?  We show that Mre11 mediates a p53-independent genome integrity pathway that is dysfunctional in approximately 10% of triple negative breast cancers. https://t.co/TYTNLV4vVI \n#WorldCancerDay  @CellReports @UNC_Lineberger https://t.co/wVSHk9MTGw","Some MIT faculty have put together a course called \"The Missing Semester of Your CS Education.\" Having looked it over a bit, it looks fantastic and will benefit data science people from non-dev backgrounds fill in a lot of gaps, too.  https://t.co/7X8CLLLg11","Giving a 2h conceptual (no calculations) intro to Bayes &amp;amp; causal inference tomorrow at local Uni. Redundant w my book, but compressed and easier. Slides are up already, and will try to record. https://t.co/FYuobSX2xp https://t.co/gsxTkXVtkc","Today is the day...finally running NoDye vs. DAPI vs. DRAQ-7 vs. 7-AAD vs. PI vs. SYTO RNASelect for FANS when running #snATAC-Seq on @10xGenomics workflow. Objective: what dye alters the chromatin the least.","Asked to write a chapter about RNA-seq counts modeling, I used #rstats bookdown to put all the material online. Includes some new workflow pieces like tximeta for import and swish for transcript level:\n\nhttps://t.co/wZiCNTEq9u","Dear #singlecell global team (#scGT), it was a very inspiring week for #scQA. @hoheyn and I would like to THANK to the @cmcginnisUCSF for his comprehensive Tweetorial on #MultiSeq. \n\nFor the Feb3/2020 #scQA/#scQC Forum (w/@hoheyn &amp;amp;\n@LGMartelotto) we'll discuss some #snATAC-Seq!","It might seem clich√© for us to focus on relationships in February, but we're looking beyond the romantic kind. ‚Å£ This is what we're planning to reflect on this month: https://t.co/dCBdXVfCqo","I have updated my post regarding bookdown Github Actions Netlify deployment to use the use_github_action() function from usethis\n\nhttps://t.co/ud9bRYP9cv","crazy what having a job you actually *like* can do!","Thanks to everyone who tuned in for  #singlecellgenomicsday! Selected slides, video, and resources are now available at https://t.co/KJpeGxemzZ\n\nFor those in a different time zone, we will rebroadcast the entire livestream next Thursday (2/6) at 9:30PM EST.","F*ck it. I want to know how much grad students pay or make around the world - we don't have that info. Anybody want to add to this thread?\n\nI am doing a PhD in microbiology at UNC-Chapel Hill. I make $32,000 USD per year before tax, and my tuition/fees are paid for.","First the chromatin opens, then the DNA methylation is lost.  \n\nThe chromatin opens because transcription factors (TFs) target those loci.\n\nTFs tell DNA methylation and nucleosomes what to do at cis-regulatory loci. Not the other way around.\n\nYour textbook needs a rewrite. https://t.co/J4LwkGqgRd","Session at #rstudioconf is about to start so here are the link to my slides https://t.co/vucMo5iEGA and the link to the livestream https://t.co/wUeZWRMX4y which more importantly is also the livestream to watch @nnstats talk. I‚Äôm on at 4:20, Namita right after at 4:45","Tagmentation with Tn5 transposase also works on DNA/RNA hybrid molecules!\nEnabling a new protocol for RNA-seq library preps.\nhttps://t.co/2PU4y7fzqO","I am working on an ebook, tentatively called Bite Size Bayes, that introduces Bayesian statistics gradually, for people with no prior stats.\n\nHere's Python notebook 2, if you want to check it out.\n\nhttps://t.co/DYfiVxdGvb\n\nR version coming soon!","Well that's a big claim there, no P53 activation https://t.co/e5wyPkE6Ga","*Flipbooks* present code step-by-step and side-by-side with output.  Now, you can build one in 5 minutes with {flipbookr}!  #rstudioconf2020 presentation, Weds 4pm\n\nFor demo:  \n1. install.packages(\"xaringan\") \n2. devtools::install_github(\"EvaMaeRey/flipbookr\") \n3. restart RStudio https://t.co/AQgTNF1PSX","Here is a simple shell script to generate a single \"meta-bam\" file by sampling from many RNA-seq libraries: https://t.co/mrHnOjYf3n https://t.co/Vtns9ViBz6","I'm teaching an introduction (to an introduction) to single-cell RNA-seq today and making use of slides that others might find useful: https://t.co/cEK19zru1S\n#scRNAseq","We wrote the review that I wished I'd had when I started my PhD in gene regulation &amp;amp; genomics! Read for our take on the past 40 yrs of enhancer biology, a def of an \"enhancer\", and how we think modern tech (like CRISPR screens) will enable the next era of understanding the genome https://t.co/4cy0UXrRns","Our latest preprint is now in the bioRxiv!   https://t.co/rLCqyd1lHg","Is there a #ggplot equivalent for making tables? Just finished a paper revision that‚Äôs over 8k words (not including the massive response letter and supplement) and the time it took me to manually create and format the tables was less than crafting the response. #rstats #science","OK, my current list is 31 scRNA-seq (analysis method) benchmarks:\nhttps://t.co/gfan8Kupcp\n\nI consolidated @seandavis12's @_lazappi_'s @JMA_Data's and went through the @GenomeBiology benchmark issue.\n\nAny others I missed? https://t.co/AEpHCWcYsS","The aim of marge is to provide an R API to HOMER for the analysis of genomic data, utilizing a tidy framework to accelerate organization and visualization of analyses https://t.co/p2SIn4q5ys","To clarify, I'm looking for a big set of normal exomes that I can download made with an exome kit that is still available. https://t.co/EryfZOL79Z","Started a table of #singlecell #platforms for ppl w/ 500-1000 cells x run. Pricing as I could find by googling from @10xgenomics @celsee_inc . Can people confirm/comment/add to this? @BettsLab @VIBTechWatch @NotWIMM @techgk_patti @EarlhamInst @Dart_DNA @DaleYuzuki @LGMartelotto https://t.co/nhM8tIAzRh","UST bets on TELL-Seq https://t.co/tl9VGKG6gY - new linked read reagent kit for Illumina sequencers from Universal Sequencing Technology @universal_seq","TWEETORIAL on our new preprint: A genome engineering shootout (!) between wild-type Cas9 and 2 different PAM-flexible CRISPR enzymes ‚Äî xCas9 and Cas9-NG ‚Äî across 3 different tasks: knock-out, CRISPRi, and CRISPRa  (1/15)\nhttps://t.co/godnUWgced https://t.co/VHjEPpFybH","Probably the most important thing business school taught me was to read 10-Ks - public company annual reports, which are required SEC filings. Companies reveal who they really are in them, more so than any news story.\n\n‚ú®üåàThis has been your free MBAüåà‚ú®","@healthstatsdude @jhollist Check out The Unix Workbench from @seankross \nhttps://t.co/sS8Cr4iQJY","#TidyTuesday 2020/04 üé∂ Spotify Songs\nAs I said before, too many options, so I came up with this monster about:\n\nTHE Güü°LDEN AGE OF HIP HOP üé§\n\nüíØ% #ggplot2 #tidyverse #rstats\n#r4ds #dataviz #infographic #rap #hiphop https://t.co/iXShPJS8Zd","[1/10] 3 yrs ago, I was 1 mo away from returning to med school when I saw a peculiar trend in sequencing data. I hesitated at first, then canceled my classes, applied to PhD, and embarked on an incredible journey. I am happy to share the fruits of my labor:https://t.co/1f9bUgzf7K https://t.co/aVjTQbBQNY","I am teaching Applied Computational Genomics (geared towards early stage graduate students) again this semester. I am slowly updating the slides and posting recordings to youtube.\nhttps://t.co/4w7sVfmGiB","I‚Äôm thrilled to announce the publication of the quantitive proteomic profiling of the Cancer Cell Line Encyclopedia. The @GygiLab in collaboration with colleagues at Novartis and the Broad we measured the proteomes of 375 cell lines from various tumor lineages (1/13) https://t.co/WXgsODMzFs","Three grad students wondered if they could grow snake venom glands in the lab. They tried, it worked, and these ‚Äúorganoids‚Äù could greatly accelerate the study of snake venom which has historically been slow and underfunded despite the huge medical need. https://t.co/WmqPZPHCLy","Our {ggfortify} #rstats journal paper has reached over 100 citations! Super excited to see that it's been useful in research. With {autoplotly}üì¶, you can even dive into the plots interactively, powered by {ggplot2} and @plotlygraphs! https://t.co/tQs1GkWFSP #dataviz #tidyverse https://t.co/xRy8QuUwde","üí°Quick R tip: The {goodpractice} package can examine your #rstats code for improvements with up to 230 checks, including:\nüîπR CMD check\nüîπtest coverage\nüîπcyclomatic complexity\nüîπcode style\nüîπimports and exports and more\n\nüì¶https://t.co/5A0miF4dMK https://t.co/tkjKzxHKAg","My first first author paper was accepted!!!! It also happens to be my first EVER paper... Despite the fact that I‚Äôve been sciencing since 2011!\n\nExperiments fail.\nProjects get dropped.\nLabs get scooped.\nShit happens.\n\nIt DOES NOT make you a bad scientist.","üéö Fab step-by-step feat. {tune}\nüèé ‚ÄúProduct Price Prediction: A Tidy Hyperparameter Tuning and Cross Validation Tutorial‚Äù by @mdancho84\nhttps://t.co/NI0Komqb5h #rstats #tidymodels https://t.co/ZcCrTT95Ij","@stchlk Excellent, so far so good. Keep me posted please.","Excellent opportunity to learn more about single cell analysis. Encouraging all the single cell analysis researchers out there to make most of this opportunity \n#singlecell @LGMartelotto @hoheyn https://t.co/ugP1WZlzwt","New Blogpost! ‚ú®\n\nDeploy your bookdown project to Netlify with Github Actions\n\nSimple step by step instructions with plenty (17 !!) pictures with arrows for easy deployment üë©‚Äçüîßüë®‚Äçüîß\n\nhttps://t.co/ud9bRYP9cv\n#rstats #bookdown https://t.co/cDdxdG1OC5","REVIEW üö®: Tumor Functional #Heterogeneity Unraveled by #scRNA-seq Technologies.  https://t.co/MhwtJKXvyO","it is a good intro book for combio https://t.co/nwu1QHCYFj free pdf !","Gene expression levels are continuous rather than binary. To access biological phenotypes along such a continuum, we developed a method to titrate gene expression using systematically attenuated CRISPR guide RNAs. Check out our work in @NatureBiotech https://t.co/nC0rjMD4jS","Does anyone else like to reverse engineer websites? \n\nLike, inspect headers to see where it's hosted, use chrome extensions to see what it's built on,  and check the network tab to see what requests are being made?\n\nThinking about writing up on it, or is this just me?","DENDRO: genetic heterogeneity profiling and subclone detection by single-cell RNA sequencing https://t.co/Rm0nZQuqXP","Tweeps of #scRNAseq and #scQA, please retweet! We have openings for 4 (yes, 4!) summer internships in computational biology at @10xGenomics! We're flexible and are willing to work with you at a different time in 2020 if summer isn't quite right for your timing too.\n1/6","I did a deep-dive into @GitHub Pages, and found it's possible to create a *really* easy way to host your own blog: no code, no terminal, no template syntax.\n\nI made \"fast_template\" to pull this together, &amp;amp; a guide showing beginners how to get blogging\nhttps://t.co/lDdKYCUO7l","What The Hell Is Going On with Vic Berger?? PILOT\n\nWe made this for TBS in 2017. Sadly, it didn't work out, but we are super proud of it and hope that you enjoy it!\n\nDirected by @DaveRaige, we explored the steel industry in my hometown of Bethlehem, PA.\n\nhttps://t.co/pKl0VlLr7w https://t.co/t47MJLkYqL","Hi scientific Twitter, \n\nI have many questions to this article. \n\nBut. \n\nWhat on earth is \"random distribution\" that they constantly refer to \n\nhttps://t.co/U1GYp84pon","Why is there no deep learning tutorial that explains things in Normal People terms. I am not taking recommendations for tutorials at this time unless your tutorial is like, \"And the rank of a tensor is how many sets of bracket thingies it has when you print it out.\"","Holy crap. If you are using R/Github for your class, you really need to use the {ghclass} package from @minebocek @rundel &amp;amp; Therese Anders. https://t.co/Cs2SqNwwRS","NIGHTMARE AFTER XMAS!\n\nSORRY/HAPPY to present our benchmarking work on #singlecell sample preservation artifacts. \nWe detected strong systematic biases related to sampling timing! \n\nMUST-READ for #scRNAseq and #scATACseq of #biobank and clinical cohorts. \n\nhttps://t.co/p4rSSu7Vdz","My old package GetoptLong which is an alternative for command-line argument parsing. It is 1. simpler and more readable, 2. easy handling default values, 3. options are directly exported as variables, 4. flexible configuration for options. https://t.co/u6IZjZ9VAI https://t.co/VFhQ47Oncu","Introduction to #Cerebro: for people who don't have any bioinformatic skills. https://t.co/XHksvqD77u","@drob @alexcookson @robinson_es https://t.co/Lki3QxuiT8","Check out my latest blog post #bioinformatics tutorial + tips on #RNAvelocity analysis (in situ) using sample #singlecell data from our latest @PNASNews paper on 10k gene #spatialtranscriptomics by #MERFISH https://t.co/XqGCN6VE8b #Rstats #rTutorial #DataScience #transcriptomics https://t.co/CCWni1IFyV","1) The GC content of exons across human chromosome 7.\n2) Mean exome read depth as function of GC content.\n\nLook at the very non-random structure across the chromosome! Shows how important it is to GC correct if you want good copy number calls from exomes. https://t.co/L6WlGKz5kO","üòçüòçüòç Highly Multiplexed Single-Cell Full-Length cDNA Sequencing of human immune cells with 10X Genomics and R2C2 | bioRxiv https://t.co/Q5znvKz6fL","@KadashKate If I may, some databases that are useful can be found here. I have used some of them when I was doing mostly cancer research. Hope this list can help.\n\nhttps://t.co/R6b4C8VrvF","Optionally hide your code in RMarkdown with code_folding https://t.co/nMKRJBTIyN #rstats","@JonTheGeek @theRcast @_ColinFay On packaging Shiny... https://t.co/oo6Ffg9xaP","Hey I'm taking a networks course with @svscarpino again, and I've got a question I'd love to crowdsource:\n\nCan you bombard me with your favorite examples of \"because we did network science on this problem, we gained insights that we *fundamentally* could not access otherwise\"?","I keep a spreadsheet for articles I want to read, talks I want to watch, or resources I want to look at. \n\nI wrote a little script to randomly choose 1-3 items off the backlog and send me a digest email, and since then I've been reading (and tweeting) a lot more! https://t.co/2N0GDBU3KX","Hallo aus @OldWorldHB! #vicfaceholes @VicBergerIV https://t.co/x6bGtgLYs0","This is some work I've been involved with that made me feel a little honest about my anti-disciplinary soul https://t.co/Y12q0qp1P4","Today, we opensource Hera-T, 10 to 100 times faster than CellRanger, consumes a very small amount of memory, uses alignment  -- not pseudo-alignment!\nhttps://t.co/iVJMX4ad50","Hey Micro peeps:: Two grad students in my lab, Chantal Guegler and Peter Culviner, developed a super easy DIY kit for rRNA depletion for RNA-seq. Works like the RiboZero kit that @illumina inexplicably stopped selling - ours is as good and way cheaper!\n\nhttps://t.co/exvRWucLec","Antivenom was discovered in 1896. Indian cobra genomic sequencing &amp;amp; transcriptomics of venom-producing glands identifies 19 ‚Äòvenom-ome-specific-targets‚Äô to enable more specific snake-bite antidotes. üêç üß¨https://t.co/5qkoY7U3iE @NatureGenet https://t.co/x5dti1iZIq","Very excited to share our pgFARM study, which describes CRISPR/Cas9-based screening of ultraconserved poison exons during in vivo tumorigenesis! Great collaboration with @aliceb_phd, just out in @NatureGenet: https://t.co/xeCGQectY3 https://t.co/xgnaHpds63","@stchlk I'm in","Instead of referring to myself as self-taught, I'm gonna start referring to myself as community-taught. \n\nThe sites, the blogs, the books, the user groups, the confs, the forums ... all community efforts that I used to learn and advance my programming and data science knowledge.","Tidylog 1.0.0 has just arrived on CRAN üéâ\n\nTidylog provides feedback for dplyr and tidyr commands, and now includes all common data-wrangling functions!\n\nblog: https://t.co/on3mFJRXZL\ngithub: https://t.co/ITPuXH9JEa\n\n#rstats @dataandme @hadleywickham https://t.co/SdaCiX9Goe","Probabilistic gene expression signatures identify cell-types from single cell #RNA-seq data https://t.co/5QcNd05Yzj\n#genomics #SingleCell #bioinformatics \nfrom Isabella Grabski &amp;amp; @rafalab https://t.co/DdflQH5a5R","Scanpy 1.4.5 is out with an overhauled website and a long-overdue first tutorial on dataset integration. Most contributions from Scanpy core (@trueflyingsheep, @ivirshup, @gokcen, @fidel_ramirez) with the \"ingest\" tool due to @koncopd.\n\nhttps://t.co/EnsNM09LMv https://t.co/gbFeLEDOA4","@bryan_howie @tangming2005 I recommend the Rosalind problems online. Try solving them in a preferred language and then try them again in another language. The statistical modeling part is interesting but I just grab publicly available data and practice pipelines whether it‚Äôs NGS, microarray‚Äôs etc.","Favorite readings on the non-profit industrial complex (besides anything from 'The Revolution Will Not Be Funded'), specifically for sharing with my coworkers at a non-profit (not necessarily socialists)?","Yanmei's paper is out today. MosaicForecast uses 30+ read-level features (+ML) to identify mosaic mutations in normal cells, where the variant allele fraction is &amp;lt;&amp;lt; what you typically see in cancer cells. A key innovation is to use haplotype phasing to generate a training set. https://t.co/FDMnizwF71","The built-in shell environment is tremendously productive, but ‚Äúsecond generation‚Äù tools can make it even more so. In https://t.co/ZGPihLlfJp I review five of these tools that I enjoy using: tmux, fzf, autojump, fd, and ag. https://t.co/QwH2O5pkZS","ü§ò From a true [hrbr]master of wrapper packages‚Ä¶\nüì¶ \"Writing Frictionless R Package Wrappers ‚Äî Building A Basic R Package\" by @hrbrmstr https://t.co/ng6ywkEjLC #rstats\nüìñ see also its bookdown companion: https://t.co/gGjmblcBM2","Do we think the NIH will accept a photo of the 1000-piece puzzle I'm almost done with in lieu of a Research Strategy section? 'cause I'm really making progress","#scQC Jan06/2020 #singlecell Question and Answer Forum (w/ @LGMartelotto and @hoheyn). \nTODAY: SAMPLE PRESERVATION\nFresh samples are optimal for scRNAseq. But what to do if samples need to be stored, transported, archived? Cold storage, crypopreservation or fixation?","Introducing BioTuring Cell Search: a completely new way to explore #singlecell RNA-seq data\n\nComing with our database of 5.8 million cells, the new Cell Search engine enables researchers to select a group of cells and search for similar populations across this entire database. https://t.co/mNmZ9RBCb2","@stephspiel I really like ComplexHeatmap because you can customize almost everything and it is well documented üë©‚Äçüíª","Ah! This very cool work (missed earlier) systematically identified genes whole KO induces sensitivity or resistance to DNA damaging agents. Shout out to this wonderful lab! @durocherlab\nhttps://t.co/MC55ouLTRL","DNA damage induced during mitosis undergoes DNA repair synthesis  https://t.co/jMENt1Do32 #bioRxiv","Trying https://t.co/T0xd0XTqC6","NEW BLOG POST: The birthday paradox puzzle: tidy simulation in R\n\nhttps://t.co/RjlhAAKH4n #rstats #tidyverse https://t.co/1vrq5rRJCA","I hate `GenomicRanges`. I just want the damn intersections WITH THE METADATA.","@WordPress This command in the terminal downloaded the whole directory. 660MB in 2 minutes.\n\nwget -r -np -nH --cut-dirs=3 -R index.html http://hostname","The best presentation on PCA that I've ever seen. I will be stealing - I mean \"liberating\" - this teaching method in the future. \n\nWe could make a great animation using this too...\n\nAlso, Bryan, @SydeakaWatson, and I are speaking at the SF RUG in Jan: https://t.co/NmWWLDeMoO https://t.co/YBt01nkLdL","Since I joined @TheDailyPilot almost four years ago, I‚Äôve attended a number of community/city meetings where the topic of homelessness has been discussed. \n\nDuring those, one phrase almost always seemed to pop up: ‚ÄúFair share.‚Äù"],[["<a href=\"https://satijalab.org/scgd/\" target=\"_blank\">https://satijalab.org/scg<\/a>","<a href=\"https://twitter.com/satijalab/status/1222901738736275458\" target=\"_blank\">https://twitter.com/satij<\/a>"],"<a href=\"https://www.nature.com/articles/s41586-020-1943-3\" target=\"_blank\">https://www.nature.com/ar<\/a>","<a href=\"https://twitter.com/brooklynevery1/status/1215406117968281600\" target=\"_blank\">https://twitter.com/brook<\/a>","<a href=\"https://www.nature.com/articles/d42859-020-00001-y\" target=\"_blank\">https://www.nature.com/ar<\/a>","<a href=\"https://youtu.be/_NEMHM1wDfI\" target=\"_blank\">https://youtu.be/_NEMHM1w<\/a>","<a href=\"https://www.biorxiv.org/content/10.1101/2020.02.03.933002v1\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","","<a href=\"https://www.cell.com/cell-reports/fulltext/S2211-1247(20)30029-2\" target=\"_blank\">https://www.cell.com/cell<\/a>","<a href=\"https://missing.csail.mit.edu/\" target=\"_blank\">https://missing.csail.mit<\/a>","<a href=\"https://speakerdeck.com/rmcelreath/bayesian-inference-is-just-counting\" target=\"_blank\">https://speakerdeck.com/r<\/a>","","<a href=\"http://mikelove.github.io/counts-model\" target=\"_blank\">http://mikelove.github.io<\/a>","","","<a href=\"https://www.hvitfeldt.me/blog/bookdown-netlify-github-actions/\" target=\"_blank\">https://www.hvitfeldt.me/<\/a>","","<a href=\"http://www.satijalab.org/scgd\" target=\"_blank\">http://www.satijalab.org/<\/a>","","<a href=\"https://twitter.com/molecularcell/status/1222548588930011137\" target=\"_blank\">https://twitter.com/molec<\/a>",["<a href=\"https://docs.google.com/presentation/d/1P9iT_TQUoxxCyvkfCLe5Y-9xBWpHh7btlP2mQ3XyNOM/edit?usp=sharingand\" target=\"_blank\">https://docs.google.com/p<\/a>","<a href=\"https://rstudio.com/conference/imperial-ballroom/\" target=\"_blank\">https://rstudio.com/confe<\/a>"],"<a href=\"https://www.pnas.org/content/early/2020/01/24/1919800117\" target=\"_blank\">https://www.pnas.org/cont<\/a>","<a href=\"https://colab.research.google.com/github/AllenDowney/BiteSizeBayes/blob/master/02_bayes.ipynb\" target=\"_blank\">https://colab.research.go<\/a>","","","<a href=\"https://github.com/Transipedia/meta-bam\" target=\"_blank\">https://github.com/Transi<\/a>","<a href=\"https://figshare.com/articles/Introduction_to_single-cell_RNA-seq_technologies/7704659\" target=\"_blank\">https://figshare.com/arti<\/a>","<a href=\"https://twitter.com/NatureRevGenet/status/1222103381306675200\" target=\"_blank\">https://twitter.com/Natur<\/a>","<a href=\"https://doi.org/10.1101/2020.01.25.919738\" target=\"_blank\">https://doi.org/10.1101/2<\/a>","",["<a href=\"https://docs.google.com/spreadsheets/d/1Gqn0eZ8oiNh8-9ovyh4D_ZsoWcKYoKTJ0C1AfrJrdRA/edit?usp=sharing\" target=\"_blank\">https://docs.google.com/s<\/a>","<a href=\"https://twitter.com/markrobinsonca/status/1220048175320440832\" target=\"_blank\">https://twitter.com/markr<\/a>"],"<a href=\"https://robertamezquita.github.io/marge/\" target=\"_blank\">https://robertamezquita.g<\/a>","<a href=\"https://twitter.com/michelebusby/status/1221865425509715968\" target=\"_blank\">https://twitter.com/miche<\/a>","","<a href=\"https://omicsomics.blogspot.com/2020/01/ust-bets-on-tell-seq.html\" target=\"_blank\">https://omicsomics.blogsp<\/a>","<a href=\"https://www.biorxiv.org/content/10.1101/2020.01.22.916064v1\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","","<a href=\"https://seankross.com/the-unix-workbench/\" target=\"_blank\">https://seankross.com/the<\/a>","","<a href=\"https://science.sciencemag.org/content/367/6476/405\" target=\"_blank\">https://science.sciencema<\/a>","<a href=\"https://github.com/quinlan-lab/applied-computational-genomics\" target=\"_blank\">https://github.com/quinla<\/a>","","<a href=\"https://www.theatlantic.com/science/archive/2020/01/how-make-snake-venom-without-any-snakes/605347/\" target=\"_blank\">https://www.theatlantic.c<\/a>","<a href=\"http://bit.ly/30OAX8b\" target=\"_blank\">http://bit.ly/30OAX8b<\/a>","<a href=\"https://cran.r-project.org/package=goodpractice\" target=\"_blank\">https://cran.r-project.or<\/a>","","<a href=\"https://buff.ly/37ltfEQ\" target=\"_blank\">https://buff.ly/37ltfEQ<\/a>","","<a href=\"https://twitter.com/hoheyn/status/1207231472668684288\" target=\"_blank\">https://twitter.com/hohey<\/a>","<a href=\"https://www.hvitfeldt.me/blog/bookdown-netlify-github-actions/\" target=\"_blank\">https://www.hvitfeldt.me/<\/a>","<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/31952776/\" target=\"_blank\">https://www.ncbi.nlm.nih.<\/a>","<a href=\"https://open.oregonstate.education/computationalbiology/\" target=\"_blank\">https://open.oregonstate.<\/a>","<a href=\"https://twitter.com/NatureBiotech/status/1216783377539510274\" target=\"_blank\">https://twitter.com/Natur<\/a>","","<a href=\"https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1922-x\" target=\"_blank\">https://genomebiology.bio<\/a>","","<a href=\"https://www.fast.ai/2020/01/16/fast_template/\" target=\"_blank\">https://www.fast.ai/2020/<\/a>","<a href=\"https://youtu.be/J5949HOaNfU\" target=\"_blank\">https://youtu.be/J5949HOa<\/a>","<a href=\"https://www.nature.com/articles/s41467-019-13624-1\" target=\"_blank\">https://www.nature.com/ar<\/a>","","<a href=\"https://rundel.github.io/ghclass/articles/ghclass.html\" target=\"_blank\">https://rundel.github.io/<\/a>","<a href=\"https://www.biorxiv.org/content/10.1101/2020.01.15.897066v1\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"https://cran.r-project.org/web/packages/GetoptLong/vignettes/GetoptLong.html\" target=\"_blank\">https://cran.r-project.or<\/a>","<a href=\"https://github.com/romanhaa/Cerebro/blob/master/README_for_biologists.md\" target=\"_blank\">https://github.com/romanh<\/a>","<a href=\"https://docs.google.com/spreadsheets/d/1pjj_G9ncJZPGTYPkR1BYwzA6bhJoeTfY2fJeGKSbOKM/edit#gid=444382177\" target=\"_blank\">https://docs.google.com/s<\/a>","<a href=\"https://jef.works/blog/2020/01/14/rna_velocity_analysis_tutorial_tips/\" target=\"_blank\">https://jef.works/blog/20<\/a>","","<a href=\"https://www.biorxiv.org/content/10.1101/2020.01.10.902361v1\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"https://public.opendatasoft.com/explore/dataset/cancer-databases/\" target=\"_blank\">https://public.opendataso<\/a>","<a href=\"https://towardsdatascience.com/ten-more-random-useful-things-in-r-you-may-not-know-about-f7c39cac7e6b\" target=\"_blank\">https://towardsdatascienc<\/a>","<a href=\"https://github.com/ThinkR-open/golem\" target=\"_blank\">https://github.com/ThinkR<\/a>","","","","<a href=\"https://www.nature.com/articles/s41586-019-1138-y\" target=\"_blank\">https://www.nature.com/ar<\/a>","<a href=\"https://github.com/bioturing/hera-t\" target=\"_blank\">https://github.com/biotur<\/a>","<a href=\"https://www.biorxiv.org/content/10.1101/2020.01.06.896837v1\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"https://www.nature.com/articles/s41588-019-0559-8\" target=\"_blank\">https://www.nature.com/ar<\/a>","<a href=\"https://www.nature.com/articles/s41588-019-0555-z\" target=\"_blank\">https://www.nature.com/ar<\/a>","","",["<a href=\"http://elbersb.com/public/posts/tidylog100/\" target=\"_blank\">http://elbersb.com/public<\/a>","<a href=\"http://github.com/elbersb/tidylog\" target=\"_blank\">http://github.com/elbersb<\/a>"],"<a href=\"https://www.biorxiv.org/content/10.1101/2020.01.05.895441v1\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"https://scanpy.readthedocs.io/en/latest/release-notes.html#version-1-4\" target=\"_blank\">https://scanpy.readthedoc<\/a>","","","<a href=\"https://twitter.com/NatureBiotech/status/1214235046955167744\" target=\"_blank\">https://twitter.com/Natur<\/a>","<a href=\"http://erick.matsen.org/2020/01/04/2nd-gen-interactive-shell.html\" target=\"_blank\">http://erick.matsen.org/2<\/a>",["<a href=\"https://buff.ly/2Fmr3Re\" target=\"_blank\">https://buff.ly/2Fmr3Re<\/a>","<a href=\"https://buff.ly/39JxeN9\" target=\"_blank\">https://buff.ly/39JxeN9<\/a>"],"","","","","<a href=\"https://www.biorxiv.org/content/10.1101/845446v1\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"https://biorxiv.org/cgi/content/short/2020.01.03.893784v1\" target=\"_blank\">https://biorxiv.org/cgi/c<\/a>","","<a href=\"http://varianceexplained.org/r/birthday-problem/\" target=\"_blank\">http://varianceexplained.<\/a>","","",["<a href=\"https://www.meetup.com/R-Users/events/267236988/\" target=\"_blank\">https://www.meetup.com/R-<\/a>","<a href=\"https://twitter.com/jaredlander/status/1196439420272070658\" target=\"_blank\">https://twitter.com/jared<\/a>"],""]],"container":"<table class=\"display fill-container\">\n  <thead>\n    <tr>\n      <th>Date<\/th>\n      <th>Handle<\/th>\n      <th>Text<\/th>\n      <th>URL<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"autoWidth":true,"columnDefs":[{"width":"70%","targets":2}],"order":[],"orderClasses":false,"rowCallback":"function(row, data) {\nvar value=data[0]; $(this.api().cell(row, 0).node()).css({'font-size':'70%'});\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'font-size':'70%'});\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'font-size':'70%'});\nvar value=data[3]; $(this.api().cell(row, 3).node()).css({'font-size':'70%'});\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'width':'500px'});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<script type="application/htmlwidget-sizing" data-for="htmlwidget-d000429013698f738f6e">{"viewer":{"width":"100%","height":800,"padding":0,"fill":false},"browser":{"width":"100%","height":800,"padding":0,"fill":false}}</script>
</body>
</html>
