---
title: "Journal Club Notes on Chromatin Accessibility"
author: Kevin Stachelek
date: '2018-09-16'
slug: journal-club-notes-on-chromatin-accessibility
categories: []
tags: []
---



<p>I’m reading an article on</p>
<p><a href="https://doi.org/10.1038/s41588-018-0175-z">Dynamic interplay between enhancer–promoter topology and gene activity</a></p>
<p>It is about chromatin accessiblity. It seems to describe a system for measuring the proximity of promoter-enhancer pairs.</p>
<p>So far I have only covered the setup of the system and a few control experiments designed to assess it’s reliability.</p>
<p>I started reading another paper on the advice of my PI:</p>
<p><a href="http://www.genesdev.org/cgi/doi/10.1101/gad.229328.113">Ubiquitously transcribed genes use
alternative polyadenylation to achieve
tissue-specific expression</a></p>
<p>I’m not too familiar with polyadenylation as a regulatory mechanism.</p>
<p>They claim that more than half of human genes use alternative cleavage and polyadenylation to generate mRNA transcripts that differ in the lengths of their 3’ untranslated regions</p>
<p>Interestingly, they claim that multi-UTR genes mostly change 3’ UTR isoform ratios to achieve tissue specificity.</p>
<p>So, instead of finding global shifts in 3’ UTR length during transformation and differentiation, they identify tissue -specific groups of multi-UTR genes that change their 3’ UTR ratios.</p>
