---
title: "Karyoploter for Somatic Copy Number Analysis"
author: "Kevin Stachelek"
date: '2018-01-06'
slug: karyoploter-for-somatic-copy-number-analysis
tags: []
categories: []
---



<p>I have recently been using <a href="https://bernatgel.github.io/karyoploter_tutorial">Karyoploter</a> for plotting somatic copy number alterations.</p>
<p>I started doing this so that I could visualize gains and losses over multiple samples in a heatmap format. I hope that developing skills plotting with karyoploter will allow me to better visualize genomic data in an R environment.</p>
<p>Unfortunately, in order to plot multiple samples I need to do some for-loop hackery.</p>
<p>First, I define a function that plots segment data for an individual sample, <code>plot_base_seg</code>. This includes the basic karyoploter function <code>kpPlotRegions</code>. I also define a counter variable, <code>tn</code> which will define spacing between samples.</p>
<pre class="r"><code>plot_base_seg &lt;- function(cov_granges, tn, genes_granges, kp, chr_select, ...){
  tr.i &lt;- 0.03
  tr.o &lt;- 0.045
  kp &lt;- kpDataBackground(kp, r0=tr.o*tn, r1=tr.o*tn+tr.i) %&gt;%
    kpAxis(ymin =-2, y = 6, cex = 0.5, r0=(tr.o*tn), r1=(tr.o*tn+tr.i)) %&gt;% 
    kpSegments(cov_granges, y0 = cov_granges$seg.mean, y1 = cov_granges$seg.mean, ymin=-5, ymax=5, r0=tr.o*tn, r1=tr.o*tn+tr.i) %&gt;% 
    kpPlotRegions(genes_granges, col=&quot;#FFAACC&quot;, r0=tr.o*tn, r1=tr.o*tn+tr.i) %&gt;% 
    kpText(genes_granges, chr=chrom(genes_granges), x=start(genes_granges), y=0,
           ymin=-6, ymax=6, labels=genes_granges$gene, col=&quot;red&quot;,
           pos=4, cex=0.5, r0=tr.o*tn, r1=tr.o*tn+tr.i) %&gt;% 
    kpText(chr=chr_select, x=10000000, y=0.2, r0=tr.o*tn, r1=tr.o*tn+tr.i, col=&quot;red&quot;, labels=unique(cov_granges$sample_id), cex=0.5)
  
}</code></pre>
<p>Then, I create a second function, <code>plot_all_to_file</code> which iterates over a <a href="https://bioconductor.org/packages/release/bioc/vignettes/GenomicRanges/inst/doc/GenomicRangesIntroduction.html">grangeslist object</a> of all samples, feeding each to <code>plot_base_seg</code> with <a href="https://github.com/tidyverse/purrr">purrr</a> which outputs a pdf heatmap.</p>
<pre class="r"><code>num_seg_granges &lt;- seq(0, length(seg_granges)-1)
plot_all_to_file &lt;- function(raw_cov_list, file_name, gene_ranges, chr_select, ...) {
  pdf(file_name)
  plot.params &lt;- getDefaultPlotParams(plot.type=1)
  plot.params$ideogramheight &lt;- 15
  plot.params$data1height &lt;- 500
  plot.params$data1inmargin &lt;- 5
  plot.params$data1outmargin &lt;- 10
  plot.params$bottommargin &lt;- 10
  plot.params$topmargin &lt;- 10
  kp &lt;- plotKaryotype(genome=&quot;hg19&quot;, plot.type = 1,plot.params = plot.params, labels.plotter = NULL, chromosomes = chr_select) %&gt;% 
    kpAddChromosomeNames(col=&quot;red&quot;)
  map2(raw_cov_list, num_seg_granges, plot_base_seg, gene_ranges, kp, chr_select)
  dev.off()
}</code></pre>
<div class="figure">
<img src="heatmap_SCNA.png" alt="Karyoploter Heatmap" />
<p class="caption">Karyoploter Heatmap</p>
</div>
