<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"/>
<script src="datatables_libs/htmlwidgets/htmlwidgets.js"></script>
<script src="datatables_libs/jquery/jquery.min.js"></script>
<link href="datatables_libs/datatables-css/datatables-crosstalk.css" rel="stylesheet" />
<script src="datatables_libs/datatables-binding/datatables.js"></script>
<script src="datatables_libs/pymjs/pym.v1.min.js"></script>
<link href="datatables_libs/dt-core/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="datatables_libs/dt-core/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="datatables_libs/dt-core/js/jquery.dataTables.min.js"></script>
<link href="datatables_libs/crosstalk/css/crosstalk.css" rel="stylesheet" />
<script src="datatables_libs/crosstalk/js/crosstalk.min.js"></script>
  <title>datatables</title>
</head>
<body style="background-color:white;">
<div id="htmlwidget_container">
  <div id="htmlwidget-849262f5c691020c8485" style="width:100%;height:800px;" class="datatables html-widget"></div>
  <script>HTMLWidgets.pymChild = new pym.Child();HTMLWidgets.addPostRenderHandler(function(){
                                setTimeout(function(){HTMLWidgets.pymChild.sendHeight();},100);
                            });</script>
</div>
<script type="application/json" data-for="htmlwidget-849262f5c691020c8485">{"x":{"filter":"none","fillContainer":true,"data":[["2018-12-31","2018-12-30","2018-12-28","2018-12-28","2018-12-26","2018-12-25","2018-12-25","2018-12-23","2018-12-22","2018-12-22","2018-12-20","2018-12-20","2018-12-19","2018-12-19","2018-12-17","2018-12-13","2018-12-12","2018-12-11","2018-12-07","2018-12-05","2018-12-04","2018-12-02","2018-12-02","2018-12-02","2018-12-01","2018-12-01","2018-12-01","2018-11-30","2018-11-29","2018-11-24","2018-11-23","2018-11-20","2018-11-18","2018-11-14","2018-11-13","2018-11-13","2018-11-13","2018-11-09","2018-11-09","2018-11-07","2018-11-06","2018-11-05","2018-11-05","2018-11-02","2018-10-31","2018-10-26","2018-10-25","2018-10-22","2018-10-18","2018-10-17","2018-10-17","2018-10-17","2018-10-17","2018-10-17","2018-10-16","2018-10-13","2018-10-13","2018-10-13","2018-10-12","2018-10-12","2018-10-06","2018-10-03","2018-10-03","2018-10-01","2018-10-01","2018-10-01","2018-10-01","2018-10-01","2018-09-29","2018-09-29","2018-09-28","2018-09-27","2018-09-27","2018-09-26","2018-09-26","2018-09-26","2018-09-25","2018-09-25","2018-09-25","2018-09-24","2018-09-24","2018-09-23","2018-09-23","2018-09-23","2018-09-17","2018-09-17","2018-09-17","2018-09-14","2018-09-13","2018-09-11","2018-09-11","2018-09-07","2018-09-07","2018-09-07","2018-09-06","2018-09-05","2018-09-05","2018-09-04","2018-09-04"],["StephdeSilva","denvercoder","djnavarro","apreshill","JennyBryan","djnavarro","alexpghayes","i_steves","palewire","timothyjacques","apreshill","AliciaOshlack","dataandme","ma_salmon","theotheredgar","fbreitw","gvwilson","ADealyVidal","Emil_Hvitfeldt","shazanfar","hyunhwanj","dennybritz","saketkc","mxwlj","KirkDBorne","nic_crane","dataandme","IAmSamFin","PhilippBayer","f2harrell","JanSkotheim","byuhobbes","NinaSchiett","laeRusers","StatGarrett","casey6r0wn","dataandme","ijlyttle","yoniceedee","RLadiesLA","candiceaman","malco_barrett","JEFworks","nlbigas","ji_timedreamer","jonmcalder","strnr","LockeData","BioMickWatson","StatGarrett","yokofakun","psmibert","michaelhoffman","NatureRevGenet","chapmanb","CompBiolPapers","biorxiv_bioinfo","KentsisResearch","teoliphant","roliepoly","sc_papers","EarlhamInst","dataandme","TiffanyTimbers","jtleek","brew_sci","10xgenomics","granvillejmath","AFrankell","JeffTownsend","OmaymaS_","biorxiv_genomic","vallens","jeremyphoward","coletrapnell","hausfath","TiffanyTimbers","strnr","nmatasci","generegulation","RemoRohs","dritoshien","strnr","mason_lab","alejagerosa","cpsievert","KVittingSeerup","dataandme","biobeth","davisjmcc","markrobinsonca","mansfieldnstuff","sajraparicio","AliciaOshlack","page_gould","SrinivasaRaoRao","mansfieldnstuff","seandavis12","gvwilson"],["I haven‚Äôt worked out my #rstats goals for the year, but a lot of people are intending to write their first package.\n\nSo here‚Äôs my temp goal: hit me up if you hit a road block. If I‚Äôve got the capacity, I‚Äôll talk you through it :)\n\nAnyone else got some time for new package devs?","I think it‚Äôs time for me to tell my full story. It might lose me some followers but so be it.","I'm seeing all these really fun \"end of year\" threads on twitter, so here's mine: a list of my 10 favourite weird #rstats creations from this year.","An #rstats #blogdown file hierarchy cheatsheet:\n\n‚îú‚îÄ archetypes &amp;lt;- edit me! \n‚îú‚îÄ config.toml &amp;lt;- edit me! \n‚îú‚îÄ content &amp;lt;- edit me! \n‚îú‚îÄ data &amp;lt;- edit me! \n‚îú‚îÄ layouts &amp;lt;- edit me! \n‚îú‚îÄ public &amp;lt;- ignore me!\n‚îú‚îÄ static &amp;lt;- use me! (png/pdf/csv/xls)\n‚îú‚îÄ themes &amp;lt;- don't touch! https://t.co/gvVA703Lwa","We are moving into the hard core tech support phase of the holidays.","A little Christmas present to myself üéÅ\nLearning Statistics with R has an internet home üè†of its very own \nhttps://t.co/uwTlvx3rOK","new blog post! everything i wish i'd known when i first started writing üñ•Ô∏è #rstats @mcmc_stan code!\n\nworks through my development progress üî® for a categorical-dirichlet model üìà for survey üì© responses https://t.co/JYrDvGX6fi","New #rstats blogpost (finally!!), in which I describe: \n\nüë©‚Äçüîß iterative testing when building plumber APIs \n‚úçÔ∏è my function-writing process\nüßô‚Äç‚ôÄÔ∏è how to control RStudio IDE terminals with the rstudioapi\nüßê package states\n\nhttps://t.co/U4xaRZoLvJ https://t.co/Kr2hvN4xUy","üéÅ #MERRYMERRY üéÅ\n\nIn the spirit of the season, your @LATdatadesk has created a new library listing all of the open-source tutorials and code examples we've published online.\n\nhttps://t.co/2Ir8cc96ix https://t.co/QQsBALEygP","So tender and mild? Is he a holy infant or a freaking chicken finger?","Finally got around to updating my #rstats #blogdown site: https://t.co/Rl9kOFs0uW\n\nüê¶ added handle to posts (@dataandme said to put a ü¶ú on it)\nüîó added \"view source\" to posts (.md OR .Rmd!)\nüñ±Ô∏è added a data download button using xfun::embed_files() (thanks @xieyihui x 2!) https://t.co/ZK9Uui37eb","Our new paper: Using Equivalent Class counts directly for testing differential transcript usage. \n\nNice work from @marekcmero\n@nadia_davidson https://t.co/C7fuJYcJLR https://t.co/JTANHlLQzZ","üéÑ¬†Miss any of the earlier #devAdvent¬†chirps? I'm stashing 'em here!\n‚ÄúüéÅ¬†devAdvent #rstats-style‚Äù\nhttps://t.co/YdAcAjrafz","Since then I've been keeping the resources of my blog post \"How to develop good #rstats packages\" more or less up-to-date: https://t.co/UxqzIEROyb https://t.co/vG1ViEdi3P","@dataandme @jdatap @krlmlr Hi, we have a couple of articles that may help: https://t.co/Iw0V1jhcOF  &amp;amp; https://t.co/x5NvZxScDU","If anyone needs a quick alignment coverage calculation + visualization on the terminal, the tool `bamcov` is now available at https://t.co/Z4ErY8XSnD https://t.co/0LyT6GoE55","Toby Hodges and I are very pleased to announce the launch of \"JavaScript versus Data Science\" at https://t.co/7Syn12gS8L. We hope you find it useful - comments, corrections, contributions, and suggestions are all very welcome.","Deactivated my Facebook profile for the first time yesterday, trying to wane off. I'm not naive to lack of privacy and living in the open doesn't faze me much, but this is too close to home for me, no pun intended.\n\nhttps://t.co/Z45Vzs9xf0","Day 7 of #packagecalendar is‚Ä¶\nüîäüî¶ gghighlight \n\nGiving some data extra emphasis during data visualizations can be tricky at best. The amazing gghighlight package lets you highlight using intuitive syntax and simple functions\n\nhttps://t.co/MKhgbtFHkf\n#rstats https://t.co/Zihh3QmMee","I've recently added a number of plotting functions to DCARS, have a look through the Installation and Usage instructions on Github!\n\nhttps://t.co/YDf2SMGo2C https://t.co/DSyAjzUhBJ","@kwbroman @JennyBryan Can this StackOverflow solution be a better way? https://t.co/VxKnHHKuVb","A Programmer's Introduction to Mathematics: https://t.co/lfv8QJhC0t - This book looks amazing. Learning pure theory has always been difficult for me, but once you add real-world examples and implementations the concepts usually become intuitive and clear.","A python package to fetch SRA metadata, download .sra mimicking the hierarchy on SRA (SRP =&amp;gt; SRX =&amp;gt; SRR). Uses aspera-client for ultra-fast downloads. Based on Jack Zhu and @seandavis12's SRAdb package: https://t.co/kfl9jAq3m0","Do you R Markdown but have collaborators that Microsoft Word? \n==&amp;gt; No: Quit playin. Go to \"Yes\".\n==&amp;gt; Yes: Check out redoc package by @noamross for a bridge between R Markdown and Microsoft Word: https://t.co/tRWlEaIvT9 #rstats ‚úçÔ∏è‚å®Ô∏è","Download Two Free eBooks (PDF):\n1) An Introduction to Statistical Learning, with Applications in R\n2) The Elements of Statistical Learning\n\nhttps://t.co/mQq3so6Xid #abdsc #BigData #DataScience #Statistics #MachineLearning #Algorithms #Rstats #Coding #DataScientists https://t.co/sanQP4oT7a","There was a point during my coding history where I was familiar with building R packages, RStudio and git, but didn't know how to contribute code via a pull request. \nThis repo is designed specifically to walk you through the process: \n\nhttps://t.co/ygWXmSawxM\n\nFeedback welcomed! https://t.co/aF7xdX7ik5","üì¶ package *and* cheatsheet for barcode-generation\n\"baRcodeR: label generation for repeatable workflows w/ biological samples\" by @ColauttiLab https://t.co/Gkzkl2qQow #rstats\nüìë https://t.co/hTcODEvWVC https://t.co/tvPcs6aa2j","This and @SherlockpHolmes's \"Modern Stats\" book are pieces of pedagogical art, created using bookdown. @russpoldrack's sprinkles on some amazing visual sugar with things like night mode, integrated search, PDF download, etc.\n\nThis is what all textbooks and course notes should be. https://t.co/WJxdFajcn6","Very good, concise tutorial on biostars on using conda and snakemake to run #bioinformatics jobs https://t.co/LEkrttdU15","I've been seeking a minimum-dependency auto email function for R.  This is what I came up with (may need modifications for Windows): https://t.co/AaIOytf2Wu  @JennyBryan #rstats","Our preprint showing how cell growth dilutes the cell cycle inhibitor Rb to trigger cell division is now out on biorxiv, read all about it https://t.co/m6CUuIFglg","Are you a research scientist? Do you experience chronic and long-lasting pain while writing code? Ask your doctor if automated testing is for you.\n\n(Side effects may include increased productivity and a mild yet addictive sense of euphoria.)\n\n https://t.co/jlO3ta8X1L","R discovery of the day:\npurrr::safely modifies function to return list of results and errors. \nVery useful for applying functions to large lists! #rstats #fieldworkpause #purrr https://t.co/RvNXqxFkdM","Thanks to everyone who came out to the November #rstats meetup and to @phdjsep and @gvegayon for presenting on R Markdown and High Performance Computing. You can find slides and materials for presentations on our GitHub: https://t.co/GpCfgA9mfF See you Dec 10! https://t.co/5XOxHPifNc","New #rstats cheatsheet today: Tidy Evaluation with #rlang! #tidyeval is a framework for doing NSE in R (and programming with the #tidyverse). Available at: https://t.co/OZVUTm5lDl @rstudio @dataandme https://t.co/wUlNLD0v83","1) Nice discussion of this paper at group meeting today. A few more thoughts: (A) How well do eRNA transcripts or enhancers cloned into plasmids with heterologous promoters (5' or 3') recapitulate endogenous activity? More systematic studies are needed to assess this https://t.co/t3lgkPeU6a","‚úÇÔ∏èits FTW...\n\"4 ways to be more efficient using RStudio's Code Snippets, w/ 11 \nexamples\" by @jozefhajnala \nhttps://t.co/qOGaH3G6rW #rstats #rstudio https://t.co/f5clSdSrsv","Hey #rstats, want to brush up on your tidyeval? Try this interactive tutorial covering symbols, quosures, !!, !!!, and more: https://t.co/BMB2XNVjMZ\n\n(feedback welcome)","@SK_convergence @brookLYNevery1 here are some links to get you started on your journey in R errors:\n\n@pjmensink\nhttps://t.co/c4YQNqWnXy\n\n@DevinPastoor &amp;amp; @noamross https://t.co/ZsTJLqaWp3\n\n@noamross\nhttps://t.co/MvaBerVoyQ","We‚Äôre delighted to be hosting @JennyBryan for this month‚Äôs meetup on November 29th (https://t.co/hT0oaTCXiX)! Come join us and @RLadiesIrvine at the USC Health Sciences campus to hear a remix of Jenny‚Äôs useR talk on code smells! üëØ‚Äç‚ôÄÔ∏èüå¥üë©‚Äçüíª","a reminder that all voting records (or lackthereof) are public info. don‚Äôt be that bitch. please exercise this right &amp;amp; vote","New #rstats blog post: I've narrowly avoided @JennyBryan setting my computer on üî• by ditching rm(list = ls()) and setwd(). But why should I use the here package when I'm already using projects? https://t.co/ndaE0dAIWo https://t.co/ba92rsXXMB","Integrating multiple #singlecell datasets across different batches, technologies, and conditions? Check out Harmony for #rstats by @ilyakorsunsky @soumya_boston for fast + efficient unified #bioinformatics analysis.\n\nPreprint: https://t.co/VEd08O1RXB\nCode: https://t.co/ySOT8NytjH https://t.co/skHWubcr5N","Our manuscript describing a 10bp mutation rate periodicity within nucleosome-covered DNA is out in @CellCellPress. [1/11]\nhttps://t.co/UqJidiatCl https://t.co/PL00QHqWbQ","This RNA-Seq tutorial is the best I've read in a long time!! Thanks authors for sharing this.üëçüëçüëç https://t.co/DvwtfUaWcG","Just wanna give another shoutout to @MilesMcBain for his work on the datapasta package! Such a neat time saver and pragmatic tool for those adhoc copy/paste operations that naturally crop up in experimental and mundane day to day data munging https://t.co/k7uKQgtPJY #rstats","TissueEnrich: Tissue-specific gene enrichment analysis | Bioinformatics | Oxford Academic https://t.co/fwLX8zXUi2\nhttps://t.co/a7A8mM3toz\nhttps://t.co/J0f0rzqzzE https://t.co/sD2FmSxiTn","\"Packages for Testing your #rstats Package\" üì¶üî®\n\nhttps://t.co/pG0ST7B3DZ\n\nRoundup of resources by @ma_salmon https://t.co/YkV2LZNkx8","@markrobinsonca @robertclab looks great!  Will try and review in more detail, though on quick scan I should think our paper would be worth mentioning! https://t.co/jQkZgogK6Q","@vivek_reds @DataSciRookie @rstudio @dataandme It is an introduction for beginners and non-programmers. I recommend Advanced R, https://t.co/Mme6BJzaLD, if you are an advanced programmer (in any language).","\"We are delighted to announce the release of gnomAD v2.1\" https://t.co/oTlzQA0Wpg  An awesome new browser; Per-gene loss-of-function constraint; Improved sample and variant filtering processes; Allele frequencies in sub-continental populations in Europe and East Asia; ...","Our latest @NYGCtech collaboration with @landau_lab, led by @AnnaEnim, @thinktank_Q and Ronan Chaligne:  GoT - Genotyping of Transcriptomes.  Combining cDNA genotyping with high-throughput scRNA-seq [1/5]. https://t.co/KVO2UPtBRR","#ASHG18 if you want to do a 3D-genome aware gene set enrichment analysis, check out BEHST. CRISPR screens not included. https://t.co/9SYX7at4xc https://t.co/SQf2X3Yg3U","Congrats to Jim Lupski on the #ASHG18 Victor A. McKusick Leadership Award! A free-to-read link to his Review on 'Mechanisms underlying structural variant formation in genomic disorders' is here: https://t.co/3t6AIkkpic https://t.co/redDjx6ZaB","@ngsstudent @malachigriffith @tangming2005 We use VarDict (https://t.co/kNKR0KoeAX) for tumor-only calling, but have also been comparing to Pisces, Octopus, FreeBayes and smcounter2 on low frequency tumor-only benchmarks, with and without UMIs. Here is the current validation work: https://t.co/i6O9ap0V4V","Algorithmic improvements for discovery of germline copy number variants in next-generation sequencing data [NEW RESULTS] https://t.co/SfnRlE4VfH","Development of a novel signature of long noncoding RNAs as a prognostic biomarker for esophageal cancer  https://t.co/chAdexB40U #biorxiv_bioinfo","Subclonal architecture, evolutionary trajectories and patterns of inheritance of germline variants in pediatric glioblastoma https://t.co/EgXykIF8yu","Fantastic project integrating all Python visualization tools into easily composable dashboard -- https://t.co/W6eKdpqoRv . This is awesome work from the PyViz team @anacondainc. #Python #visualization","Great visual work by @nytgraphics @nytimes üì∞ https://t.co/jtqQccN2b4","CellMarker: a manually curated resource of cell markers in human and mouse. https://t.co/ZQhpkMxXel https://t.co/IoK0LiKOfw","Do you want to discover the unknown functions of alternatively spliced transcripts in the human genome?Join the @WHaerty Group to investigate vertebrate development originating from the ADAMTS genes. \n\nhttps://t.co/gsCjthry0i\n\n#PhD #bioinformatics #genetics #transcriptomics","üñ§ Must-read for anyone who writes docs (so, everyone, right?)\n\"What nobody tells you about documentation\" ‚úçÔ∏è @evildmp \nhttps://t.co/8Pk67yUgyC ht @JennyBryan https://t.co/gm6S8CUWZF","After 3 years, I think we are converging on a solid course to introduce our @UBCMDS students to the Data Science tools/software stack (e.g., Jupyter, Git &amp;amp; @github, @rstudio , Rmd, etc) and are now ready to share the materials: https://t.co/wSqrKkU5Se","We've been working to remove and reduce barriers for anyone in high school on up to get into data science. The first part of our new effort is now up! Chromebook Data Science - a MOOC series on @leanpub. \n\nhttps://t.co/3jOAhpBy8U https://t.co/hASXwPUWQJ","üéâ New formula flowcraft in Brewsci/bio for Linux and macOS!\n‚ÑπÔ∏è Nextflow pipeline assembler for genomics\nüç∫ brew install brewsci/bio/flowcraft\nüè° https://t.co/aPjzD6Abc7\nüî¨ https://t.co/aaJgJTeT2y\nüêß https://t.co/lyKHNJu5CN #bioinformatics #flowcraft @ines_cim @nextflowio","We are excited to announce the launch of the Chromium Single Cell ATAC Solution! The new solution enables researchers to reveal chromatin accessibility cell by cell with #singlecell #epigenomics #ATACseq. Learn more here: https://t.co/JNlCSZr2cS","Excited to share my blog post about how I used #rstats and googlesheets to make a reminder bot ü§ñ to automate away the boring parts of organising our (journal club) meetings. This has been a lifesaver for me, so I hope it's helpful for others too!\nhttps://t.co/ixQx1niKqt","Pharmagenomic landscapes of 462 patient-derived pan-cancer models: the biggest effort yet to predict patient response from primary tumour derived cultures out this week.\n\nhttps://t.co/mQ9Gh6ah77 https://t.co/CRWw65wCAU","Here‚Äôs what I said in my two minutes on stage @CellSymposium #TCGA Legacy meeting #CSTCGA: https://t.co/uNnWNG58Wb","New Blog Post:\nHandling R errors the rlang way: custom conditions, subclasses and more!\nhttps://t.co/zQbY2mlWZ9\n\n#rstats #tidyverse #rlang https://t.co/6ZMAMcI4Kz","Local epigenomic state cannot discriminate interacting and non-interacting enhancer-promoter pairs with high accuracy  https://t.co/tZWyfqo1Ly #biorxiv_genomic","@markrobinsonca Sounds neat, I hadnt heard of Github Classroom. Reminds me of teaching Python a few of years ago and had one assignment be a script that corrects and grades assignments. At the end an assignment was to use Github API to get data and analyse hand-in times: https://t.co/Uyr7qPokhA","Proud to be launching \"Introduction to Machine Learning for Coders\" today. If you're looking for a modern, code-first, opinionated introduction to machine learning, I think you might like this.\nhttps://t.co/lJFoX6LxMj https://t.co/PxuEAU0c4V","Want to learn gene regulatory networks from single-cell RNA-seq data? From an information theory point of view, you're going to need \"coupled\" measurements. Our new pre-print out today, led by @Xiaojie_Qiu. Collab with @sreeramkannan:  https://t.co/buwj78ThpY","After a ton of work our @CarbonBrief interactive temperature map is finally done! Browse any location in the world to see how the climate has changed in the past and will change in the future. Data for any location can be downloaded in a csv. https://t.co/PydQbSuUou https://t.co/0VWflauq1h","Poll for the #python #datascience folks out there! What do you use for plotting on a regular basis? \n\ncc @ctitusbrown @scientificbruno @billdoesphysics @Dr_CMingarelli @katyhuff @thecarpentries @SciPyConf #rstats (some of you also code in Python)","Misunderstood parameter of NCBI BLAST impacts the correctness of bioinformatics workflows https://t.co/gtbYgY5y4i Seems the importance of this can't be understated: 1/n https://t.co/7CwIFbFhfe","We are hiring! Join our team as a Data Wrangler to support the Ellison Institute's mission to transforme medicine https://t.co/XFek531XU7 #rstats #bigdata #Datascience #DataAnalysis","The interaction landscape between transcription factors and the nucleosome [Jussi Taipale &amp;amp; Co] https://t.co/V9UjjfV2Pr Explored interactions between the nucleosome and 220 TFs -&amp;gt; Many TFs have preferred binding site locations in nucleosome; some of them are periodic #TFbinding https://t.co/256Evukr8p","Excited to share that @USC &amp;amp; @USCDornsife established Quantitative &amp;amp; Computational Biology as new section of @USCBioSci with independent hiring, 10 primary &amp;amp; 15 joint faculty, 55 PhD &amp;amp; 25 undergrad students. I will chair our section, assisted by Michael Waterman. #futureofbiology","We updated the protocol of random displacement amplification sequencing (RamDA-seq 1.1), the first full-length total RNA-sequencing method for single cells. https://t.co/0Jhr4fdyx6  #SingleCell #RNAseq","@sminot @bioinformer @iprophage @fstrozzi @WDL_dev @TobiasSchraink @PaoloDiTommaso @linsalrob @aylwyn_scally have you seen @thejmazz's post? A few years old but still good. https://t.co/oYAvKDcoON\n\nit pushed us to give @nextflowio a try. happy so far.","RNA takes over DNA repair! Pryor et al. show that RNA is routinely incorporated during the repair of DNA double-strand breaks through the mammalian nonhomologous end‚Äìjoining (NHEJ) pathway (Inc. V(D)J recombination and Cas9-induced genome engineering) https://t.co/zXhpRxwhLH","How do I get \"real life data\" experience with SQL queries? Are there public/open databases that are best accessed this way? Maybe data science education gurus @kierisi, @dataandme and @BecomingDataSci know the answer? (or can point out any obvious solution I missed)","üì£ Announcement time: I'm joining @rstudio as a software engineer! üéâ\n\nFor at least the next 5-6 months, I'll be expanding upon https://t.co/mkiZkaqefW and maintaining the @plotlygraphs for #rstats package! üìöüìäüìà","Our major update to IsoformSwitchAnalyzeR is now up on @bioconductor. The improvements are centered around robustness, speed and usability. Recommend for everybody https://t.co/QYBPbvcMpH #bioinformatics #rnaseq #alternativesplicing #rstats","üò≤ {remedy} has more tricks than I remembered (+ keyboard shortcut recs to boot!)\n\"remedy: RStudio Addins to Simplify Markdown Writing\" \nhttps://t.co/0BSJ9MF3fl via @thinkR_fr https://t.co/q6yf7XcNlV","our informal beginners python club is looking for projects/datasets to test our skills - any ideas? we all have a decent genetics background and were hoping to make something useful (or amusing)üêç","Delighted to share our latest work with Raghd Rostom, @YuanhuaHuang, @OliverStegle, Sarah Teichmann et al. Cardelino:‚Äã ‚ÄãIntegrating whole exomes and single-cell transcriptomes to reveal phenotypic impact of somatic variants https://t.co/i4npjUnh60 1/9","'v2' of the Angelo's and @CSoneson's (scRNA) clustering comparison manuscript is published!\n\nhttps://t.co/xXqWj3PNbF\n\nResearch moves fast with modern publishers like @F1000Research and fast, constructive reviews from @JEFworks and @trashystats https://t.co/VhTWm7aDo0",".@GavinNewsom https://t.co/MjBojgPKpM","For anyone interested in the biology of single cell genomes, see our just posted preprint on 40,000 genomes sequenced with an open source new method, DLP+. Clonal merging, negative selection and replication states as examples. https://t.co/4rT7aDhJ9Q","Our new paper just published in @GigaScience. \nXimmer: A system for improving accuracy and consistency of CNV calling from exome data\nCongratulations to Simon Sadedin.\nhttps://t.co/WIXr4TdIb1","I‚Äôm teaching my PhD stats class again this fall ‚Äî all my materials will be available as open materials at https://t.co/BmDJuFF1Ed, once I start lecturing next Tuesday ‚Äî and I‚Äôm translating my #rstats code to #tidyverse","TIL how to build a flowchart from a complex #GATK #WDL workflow using the very cool #DiagrammeR #rstats package by @riannone... so inordinately proud of this rough and dirty solution! üôÇ code here: https://t.co/RdHK4WktPZ https://t.co/SJwYi6R69I","Woah hadn't listened to the water temple theme in Ocarina of Time in a long time... dang that's good creepy stuff","New tooling from @StatGenDan:\n\"The Mega2R package: R tools for accessing and processing genetic data in common formats\": convert from different genetic data formats into input for over 40 commonly-used genetic analysis programs.\nhttps://t.co/4Sa384Ed6l","We are pleased to announce Version 0.2 (ish) of \"JavaScript versus Data Science\". It's still point-form notes with a lot of FIXMEs, but we would be grateful for feedback - please check it out at https://t.co/cJ3VZHWdQq. #rse18 @thecarpentries @tbyhdgs @dabeaz"],["","","","","","<a href=\"http://learningstatisticswithr.com\" target=\"_blank\">http://learningstatistics<\/a>","","<a href=\"https://irene.rbind.io/post/iterative-testing-plumber/\" target=\"_blank\">https://irene.rbind.io/po<\/a>","<a href=\"https://github.com/datadesk/tutorials\" target=\"_blank\">https://github.com/datade<\/a>","","<a href=\"https://alison.rbind.io/\" target=\"_blank\">https://alison.rbind.io/<\/a>","<a href=\"https://www.biorxiv.org/content/early/2018/12/19/501106\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"https://buff.ly/2BtaMHY\" target=\"_blank\">https://buff.ly/2BtaMHY<\/a>",["<a href=\"https://masalmon.eu/2017/12/11/goodrpackages/\" target=\"_blank\">https://masalmon.eu/2017/<\/a>","<a href=\"https://twitter.com/rOpenSci/status/1075154534085738496\" target=\"_blank\">https://twitter.com/rOpen<\/a>"],["<a href=\"https://db.rstudio.com/best-practices/dashboards/\" target=\"_blank\">https://db.rstudio.com/be<\/a>","<a href=\"https://db.rstudio.com/best-practices/deployment/\" target=\"_blank\">https://db.rstudio.com/be<\/a>"],"<a href=\"https://github.com/fbreitwieser/bamcov\" target=\"_blank\">https://github.com/fbreit<\/a>","<a href=\"https://github.com/software-tools-in-javascript/js-vs-ds\" target=\"_blank\">https://github.com/softwa<\/a>","<a href=\"https://buff.ly/2BcFkxg\" target=\"_blank\">https://buff.ly/2BcFkxg<\/a>","<a href=\"https://github.com/yutannihilation/gghighlight\" target=\"_blank\">https://github.com/yutann<\/a>","<a href=\"https://github.com/shazanfar/DCARS\" target=\"_blank\">https://github.com/shazan<\/a>","<a href=\"https://stackoverflow.com/a/420172\" target=\"_blank\">https://stackoverflow.com<\/a>","<a href=\"https://pimbook.org/\" target=\"_blank\">https://pimbook.org/<\/a>","<a href=\"https://github.com/saketkc/pysradb\" target=\"_blank\">https://github.com/saketk<\/a>","<a href=\"https://github.com/noamross/redoc\" target=\"_blank\">https://github.com/noamro<\/a>","<a href=\"http://bit.ly/1llYVcH\" target=\"_blank\">http://bit.ly/1llYVcH<\/a>","<a href=\"https://github.com/thisisnic/first-contributions\" target=\"_blank\">https://github.com/thisis<\/a>",["<a href=\"https://buff.ly/2KLxcIz\" target=\"_blank\">https://buff.ly/2KLxcIz<\/a>","<a href=\"https://buff.ly/2KHSFC3\" target=\"_blank\">https://buff.ly/2KHSFC3<\/a>"],"<a href=\"https://twitter.com/russpoldrack/status/1067467984422105088\" target=\"_blank\">https://twitter.com/russp<\/a>","<a href=\"https://www.biostars.org/p/335903/\" target=\"_blank\">https://www.biostars.org/<\/a>","<a href=\"http://github.com/harrelfe/rscripts/blob/master/ssmtp.r\" target=\"_blank\">http://github.com/harrelf<\/a>","<a href=\"http://disq.us/t/38qjs6b\" target=\"_blank\">http://disq.us/t/38qjs6b<\/a>","<a href=\"https://standage.github.io/the-joy-and-art-of-automated-testing.html\" target=\"_blank\">https://standage.github.i<\/a>","","<a href=\"https://github.com/laeRusers/presentations\" target=\"_blank\">https://github.com/laeRus<\/a>","<a href=\"https://www.rstudio.com/resources/cheatsheets/\" target=\"_blank\">https://www.rstudio.com/r<\/a>","<a href=\"https://twitter.com/casey6r0wn/status/1059900576849453059\" target=\"_blank\">https://twitter.com/casey<\/a>","<a href=\"https://buff.ly/2PmOdP0\" target=\"_blank\">https://buff.ly/2PmOdP0<\/a>","<a href=\"https://ijlyttle.shinyapps.io/tidyeval\" target=\"_blank\">https://ijlyttle.shinyapp<\/a>",["<a href=\"https://methodsblog.com/2018/01/26/r-errors/#more-6327\" target=\"_blank\">https://methodsblog.com/2<\/a>","<a href=\"https://github.com/noamross/zero-dependency-problems/issues/7\" target=\"_blank\">https://github.com/noamro<\/a>","<a href=\"https://github.com/noamross/zero-dependency-problems/blob/master/misc/stack-overflow-common-r-errors.md\" target=\"_blank\">https://github.com/noamro<\/a>"],"<a href=\"https://www.meetup.com/rladies-la/events/255876103/\" target=\"_blank\">https://www.meetup.com/rl<\/a>","","<a href=\"https://malco.io/2018/11/05/why-should-i-use-the-here-package/\" target=\"_blank\">https://malco.io/2018/11/<\/a>",["<a href=\"https://www.biorxiv.org/content/early/2018/11/04/461954\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"https://github.com/immunogenomics/harmony\" target=\"_blank\">https://github.com/immuno<\/a>"],"<a href=\"https://authors.elsevier.com/a/1X~jBL7PXUj6c\" target=\"_blank\">https://authors.elsevier.<\/a>","<a href=\"https://hbctraining.github.io/DGE_workshop/\" target=\"_blank\">https://hbctraining.githu<\/a>","<a href=\"https://github.com/MilesMcBain/datapasta\" target=\"_blank\">https://github.com/MilesM<\/a>",["<a href=\"https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty890/5140218\" target=\"_blank\">https://academic.oup.com/<\/a>","<a href=\"http://tissueenrich.gdcb.iastate.edu/\" target=\"_blank\">http://tissueenrich.gdcb.<\/a>","<a href=\"https://bioconductor.org/packages/release/bioc/html/TissueEnrich.html\" target=\"_blank\">https://bioconductor.org/<\/a>"],"<a href=\"https://itsalocke.com/blog/packages-for-testing-your-r-package/\" target=\"_blank\">https://itsalocke.com/blo<\/a>","<a href=\"https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0734-x\" target=\"_blank\">https://genomebiology.bio<\/a>","<a href=\"https://adv-r.hadley.nz/\" target=\"_blank\">https://adv-r.hadley.nz/<\/a>","<a href=\"https://macarthurlab.org/2018/10/17/gnomad-v2-1/\" target=\"_blank\">https://macarthurlab.org/<\/a>","<a href=\"https://www.biorxiv.org/content/early/2018/10/16/444687\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"http://behst.hoffmanlab.org/\" target=\"_blank\">http://behst.hoffmanlab.o<\/a>","<a href=\"https://rdcu.be/9pVn\" target=\"_blank\">https://rdcu.be/9pVn<\/a>",["<a href=\"https://github.com/AstraZeneca-NGS/VarDictJava\" target=\"_blank\">https://github.com/AstraZ<\/a>","<a href=\"https://github.com/bcbio/bcbio_validations/tree/master/somatic-lowfreq\" target=\"_blank\">https://github.com/bcbio/<\/a>"],"<a href=\"http://dlvr.it/QnDLZp\" target=\"_blank\">http://dlvr.it/QnDLZp<\/a>","<a href=\"http://biorxiv.org/cgi/content/short/441568v1\" target=\"_blank\">http://biorxiv.org/cgi/co<\/a>","<a href=\"https://www.biorxiv.org/content/early/2018/10/12/434241\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"https://panel.pyviz.org/\" target=\"_blank\">https://panel.pyviz.org/<\/a>","<a href=\"https://www.nytimes.com/interactive/2018/10/12/us/map-of-every-building-in-the-united-states.html\" target=\"_blank\">https://www.nytimes.com/i<\/a>","<a href=\"http://dlvr.it/QmTkN8\" target=\"_blank\">http://dlvr.it/QmTkN8<\/a>","<a href=\"https://okt.to/K7fK0z\" target=\"_blank\">https://okt.to/K7fK0z<\/a>","<a href=\"https://buff.ly/2Dh1hQ1\" target=\"_blank\">https://buff.ly/2Dh1hQ1<\/a>","<a href=\"https://github.com/UBC-MDS/public/tree/master/courses/521_platforms\" target=\"_blank\">https://github.com/UBC-MD<\/a>","<a href=\"http://jhudatascience.org/chromebookdatascience/cbds.html\" target=\"_blank\">http://jhudatascience.org<\/a>",["<a href=\"https://github.com/assemblerflow/flowcraft\" target=\"_blank\">https://github.com/assemb<\/a>","<a href=\"https://github.com/brewsci/homebrew-bio\" target=\"_blank\">https://github.com/brewsc<\/a>","<a href=\"http://linuxbrew.sh\" target=\"_blank\">http://linuxbrew.sh<\/a>"],"<a href=\"https://bit.ly/2OweHMq\" target=\"_blank\">https://bit.ly/2OweHMq<\/a>","<a href=\"https://www.granvillematheson.com/post/reminder-bot/\" target=\"_blank\">https://www.granvillemath<\/a>","<a href=\"https://www.nature.com/articles/s41588-018-0209-6.pdf\" target=\"_blank\">https://www.nature.com/ar<\/a>","","<a href=\"https://www.onceupondata.com/2018/09/28/handling-r-errors/\" target=\"_blank\">https://www.onceupondata.<\/a>","<a href=\"http://biorxiv.org/cgi/content/short/420372v1\" target=\"_blank\">http://biorxiv.org/cgi/co<\/a>","<a href=\"https://github.com/vals/scilife-python-course\" target=\"_blank\">https://github.com/vals/s<\/a>","<a href=\"http://www.fast.ai/2018/09/26/ml-launch/\" target=\"_blank\">http://www.fast.ai/2018/0<\/a>","<a href=\"https://www.biorxiv.org/content/early/2018/09/25/426981\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"https://www.carbonbrief.org/mapped-how-every-part-of-the-world-has-warmed-and-could-continue-to-warm\" target=\"_blank\">https://www.carbonbrief.o<\/a>","","<a href=\"https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty833/5106166\" target=\"_blank\">https://academic.oup.com/<\/a>","<a href=\"https://usc.wd5.myworkdayjobs.com/ExternalUSCCareers/job/Los-Angeles-CA---Health-Sciences-Campus/Data-Wrangler_REQ20065748\" target=\"_blank\">https://usc.wd5.myworkday<\/a>","<a href=\"https://www.nature.com/articles/s41586-018-0549-5\" target=\"_blank\">https://www.nature.com/ar<\/a>","","<a href=\"https://bit.riken.jp/protocols/ramda-seq/\" target=\"_blank\">https://bit.riken.jp/prot<\/a>","<a href=\"https://jmazz.me/blog/NGS-Workflows\" target=\"_blank\">https://jmazz.me/blog/NGS<\/a>","<a href=\"http://science.sciencemag.org/content/361/6407/1126\" target=\"_blank\">http://science.sciencemag<\/a>","","<a href=\"https://plotly-book.cpsievert.me/\" target=\"_blank\">https://plotly-book.cpsie<\/a>","<a href=\"http://goo.gl/GyBnxj\" target=\"_blank\">http://goo.gl/GyBnxj<\/a>","<a href=\"https://buff.ly/2MmOz1H\" target=\"_blank\">https://buff.ly/2MmOz1H<\/a>","","<a href=\"https://www.biorxiv.org/content/early/2018/09/10/413047\" target=\"_blank\">https://www.biorxiv.org/c<\/a>",["<a href=\"https://goo.gl/SZhMja\" target=\"_blank\">https://goo.gl/SZhMja<\/a>","<a href=\"https://twitter.com/markrobinsonca/status/1022471565299122176\" target=\"_blank\">https://twitter.com/markr<\/a>"],"","<a href=\"https://www.biorxiv.org/content/early/2018/09/06/411058\" target=\"_blank\">https://www.biorxiv.org/c<\/a>","<a href=\"https://academic.oup.com/gigascience/advance-article/doi/10.1093/gigascience/giy112/5091801\" target=\"_blank\">https://academic.oup.com/<\/a>","<a href=\"https://osf.io/q6wnd\" target=\"_blank\">https://osf.io/q6wnd<\/a>","<a href=\"https://github.com/sraorao/wdl_flow/blob/master/draw_flowchart_from_wdl_script.R\" target=\"_blank\">https://github.com/sraora<\/a>","","<a href=\"http://bit.ly/2ChF8AJ\" target=\"_blank\">http://bit.ly/2ChF8AJ<\/a>","<a href=\"https://software-tools-in-javascript.github.io/js-vs-ds/\" target=\"_blank\">https://software-tools-in<\/a>"]],"container":"<table class=\"display fill-container\">\n  <thead>\n    <tr>\n      <th>Date<\/th>\n      <th>Handle<\/th>\n      <th>Text<\/th>\n      <th>URL<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"autoWidth":true,"columnDefs":[{"width":"70%","targets":2}],"order":[],"orderClasses":false,"rowCallback":"function(row, data) {\nvar value=data[0]; $(this.api().cell(row, 0).node()).css({'font-size':'70%'});\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'font-size':'70%'});\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'font-size':'70%'});\nvar value=data[3]; $(this.api().cell(row, 3).node()).css({'font-size':'70%'});\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'width':'500px'});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<script type="application/htmlwidget-sizing" data-for="htmlwidget-849262f5c691020c8485">{"viewer":{"width":"100%","height":800,"padding":0,"fill":false},"browser":{"width":"100%","height":800,"padding":0,"fill":false}}</script>
</body>
</html>
